setup.py

```diff
@@ -24,6 +24,7 @@
         'requests>=2.20.0',
         'dataclasses>=0.8',
         'cyclonedx-python-lib==2.7.1',
+        'spdx-tools>0.7.1',
```

Comment:
Freeze the version, plz

Reply:
As mentioned above, the code depends on an unreleased version of spdx-tools, so I can't freeze the version yet.